Solutions/Trend Micro Cloud App Security/Data Connectors/TerndMicroCAS_API_FunctionApp.json

```diff
@@ -0,0 +1,121 @@
+{
+	"id": "TrendMicroCAS",
+	"title": "Trend Micro Cloud App Security",
+	"publisher": "Trend Micro",
+	"descriptionMarkdown": "The [Trend Micro Cloud App Security](https://www.trendmicro.com/en_be/business/products/user-protection/sps/email-and-collaboration/cloud-app-security.html) data connector provides the capability to retrieve security event logs of the services that Cloud App Security protects and more events into Azure Sentinel through the Log Retrieval API. Refer to API [documentation](https://docs.trendmicro.com/en-us/enterprise/cloud-app-security-integration-api-online-help/supported-cloud-app-/log-retrieval-api/get-security-logs.aspx) for more information. The connector provides ability to get events which helps to examine potential security risks, analyze your team's use of collaboration, diagnose configuration problems and more.",
+	"additionalRequirementBanner": ">This data connector depends on a parser based on a Kusto Function to work as expected [**TrendMicroCAS**](https://aka.ms/sentinel-TMCAS-parser) which is deployed with the Azure Sentinel Solution.",
```

Comment:
Update short link for parser as **https://aka.ms/sentinel-TrendMicroCAS-parser**

Reply:
Done